enhancements/security/openshift-image-policy.md

```diff
@@ -0,0 +1,359 @@
+
+#### Transitioning unsigned-release testing
+
+There is a transitional cost to the additional step (`internalTestingImagePolicy` for installs, `overrides` for updating older clusters) needed to run an unsigned release image for internal testing.
+But that transitional cost is only expected to affect internal use-cases; external users should always be using signed releases.
+
+#### Restricted-network use
+
+Disconnected/restricted-network use is out of scope for this enhancement, so external clusters running `TechPreviewNoUpgrade` and getting this feature may also need the `internalTestingImagePolicy` or `overrides` workaround.
+It's possible that an ImageDigestMirrorSet like this would be sufficient:
+
+```yaml
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
+metadata:
+  name: openshift-release-payload
+spec:
+  imageDigestMirrors:
+  - mirrors:
+    - example.com/my-mirror/whatever
+    source: quay.io/openshift-release-dev/ocp-release
+```
+
+In any case, it's out of scope for a reason, but until it gets picked up, it might make things more difficult for `TechPreviewNoUpgrade` disconnected/restricted-network clusters.
+
+##### Mirror registries that do not support Sigstore signatures
+
+As a subset of the restricted-network use case, when a user has an ImageDigestMirrorSet set to point at a mirror, and the canonical quay.io is either not reachable, or [`mirrorSourcePolicy: NeverContactSource`][ImageDigestMirrorSet] is set, the mirror registry will need to allow Sigstore signatures in order for CRI-O to be able to retrieve signatures.
+Quay support for Sigstore is [new and ongoing][PROJQUAY-3400], so it's likely that there are other mirror registries in use where Sigstore hosting is not yet possible.
+But again, restricted-network use is out of scope for this initial policy, and once it comes into scope, mirror support will be a generic issue for all ClusterImageProfile and ImageProfile users, and will not be unique to OpenShift's internal policies.
+
+#### Security
+
+How will security be reviewed and by whom?
+
+I have no idea; suggestions welcome :).
+
+There will be a gap during install before the ClusterImagePolicy has been created and rolled out to nodes, during which unsigned `quay.io/openshift-release-dev/ocp-release` images might run.
+But we do not perform any in-cluster signature checking during installs today; they are secured by a trusted installer using a baked-in release image digest.
+Trust in the installer is either transitive via trust in the hosting site from which the installer was downloaded, or via mirror signatures like [this one][4.15.0-multi-mirror-signature].
+
+#### User experience
+
+How will UX be reviewed and by whom?
+
+I expect "images are failing to meet the relevant image policy/policies" to be something with a solid user experience, regardless of whether OpenShift defines its own image policy, or leaves image policies entirely up to users.
```

Comment:
I'm expecting this to be surfaced via some kind of `CreateContainerConfigError`-like state where the Pod stays `Pending`. That hardly has solid experience especially for platform workloads, and even more so for internal kinds like "`Job` that extracts payload content". 

This would make the experience similar to e.g. the old bugs where custom SCCs apply to platform workloads, which does not sound great?

Reply:
I'd like the user experience to be a concise message directly from `oc` or, at a minimum, reported through ClusterVersion status and `oc adm update status` .